src/XMakeTasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -361,12 +368,6 @@ using System.Reflection%3b
         <_ExplicitReference Include="$(FrameworkPathOverride)\mscorlib.dll" />
     </ItemGroup>
 
-    <!-- Import design time targets for Roslyn Project System. These are only available if Visual Studio is installed. -->
-    <PropertyGroup>
-       <CSharpDesignTimeTargetsPath Condition="'$(CSharpDesignTimeTargetsPath)'==''">$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed\Microsoft.CSharp.DesignTime.targets</CSharpDesignTimeTargetsPath>
-    </PropertyGroup>
-    <Import Project="$(CSharpDesignTimeTargetsPath)" Condition="'$(CSharpDesignTimeTargetsPath)' != '' and Exists('$(CSharpDesignTimeTargetsPath)')" />
```

Comment:
This means that the design time targets won't be able to override stuff in Microsoft.Common.targets. Is that alright?